projects/workflows-creator/src/lib/builder/builder.component.ts

```diff
@@ -237,10 +243,12 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
     });
     this.updateState(item.element.node, item.element.inputs);
     // TODO: to be refactored
-    if (this.eventGroups[0].children?.length) {
-      this.showElseBlock =
-        item.element.node.state.get('value') !== ValueTypes.AnyValue;
-    }
+    this.elseBlockHidden =
+      this.eventGroups[0].children?.length === 1 &&
+      this.eventGroups[0].children[0].node.getIdentifier() ===
```

Comment:
?

Reply:
To hide the else block when the trigger is "Column changes to anything" and there are no other conditions.